crates/chronicle/src/bootstrap/config.rs

```diff
@@ -127,3 +153,33 @@ address = "{}"
 
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+
+    use super::{SecretConfig, ValidatorConfig};
+
+    use url::Url;
+
+    #[test]
+    fn secret_config_custom_debug() {
+        let secret_config = SecretConfig {
+            path: "some_path".into(),
+        };
+
+        insta::assert_debug_snapshot!(secret_config, @r###"
+        SecretConfig {
+            path: "***SECRET***",
```

Comment:
Sorry I didn't catch this before but would it be better to log this? The path itself isn't secret and can help point to config issues.

Reply:
Agreed, very little more annoying than 'file not found' without mentioning the file if not absolutely necessary